src/listview/row/mod.rs

```diff
@@ -6,8 +6,8 @@
 //!
 //! Views implement Autolayout, which enable you to specify how things should appear on the screen.
 //!
-//! ```rust,no_run
-//! use cacao::color::rgb;
+//! ```rust,compile_fail
```

Comment:
It is unclear to me why we expect this example to not compile?

Reply:
Fixed. I wasn't sure if I wanted `compile_fail` or `ignore` when I was starting this. That one was left over.